README.md

```diff
@@ -1,14 +1,15 @@
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
+[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/AY2425S1-CS2103T-W12-3/tp/actions)
 
 ![Ui](docs/images/Ui.png)
 
-* This is **a sample project for Software Engineering (SE) students**.<br>
+* This is **a team project for Software Engineering (SE) students from NUS**.<br>
   Example usages:
   * as a starting point of a course project (as opposed to writing everything from scratch)
   * as a case study
-* The project simulates an ongoing software project for a desktop application (called _AddressBook_) used for managing contact details.
+* The project simulates an ongoing software project for a desktop application (called _UniLink_) used for managing contact details of a University student.
```

Comment:
Maybe dumb question but are we still simulating an ongoing software project, if this is actually an ongoing software project? Maybe we should delete the line?